.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["es2015", "stage-1", "react"],
+  "presets": ["./packages/material-ui-babel-preset-es2015", "stage-1", "react"],
```

Comment:
I'm not sure about putting this preset under the packages folder. That's not something we want to publish.
What about using a `tools` folder like react-router?